opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/Ctags.java

```diff
@@ -136,22 +134,31 @@ public void close() {
         }
     }
 
-    private void initialize() throws IOException {
-        ProcessBuilder processBuilder;
-        List<String> command = new ArrayList<>();
+    /**
+     * Gets the command-line arguments used to run ctags.
+     * @return a defined (immutable) list
+     */
+    public List<String> getArgv() {
+        initialize();
+        return Collections.unmodifiableList(command);
+    }
+
+    private void initialize() {
+        env.validateUniversalCtags();
```

Comment:
is there a point of continuing if this returns False ?

Reply:
I am thinking if we should be more strict w.r.t. ctags validation and just terminate the indexer right in the beginning if no valid ctags implementation is found.

Reply:
Also, the work done by `initialize()` is repeated in every worker which seems unnecessary given all of them will share the same configuration/argv.

Reply:
`Indexer` is already strict when ctags validation fails.

I probably should have added a call to `validateUniversalCtags()` to opengrok-web instead of putting it into `Ctags.initialize()`, since my intention anyway was to ensure `Ctags` operates for `--webappCtags on` the same as during indexing.

Reply:
Regarding the repetition of `initialize()`, we already minimize `Ctags` construction with `CtagsObjectFactory`, so a small bit of redundant initialization seemed harmless.